### PR TITLE
Feat/rl 7000615707/add video to oam

### PR DESCRIPTION
**Pull request by RyanCrux**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

Added video to homepage AND to news & events

## Related Tickets & Documents
- Monday Task: https://blackberg-group.monday.com/boards/1931719392/pulses/7000615707

## QA Instructions, Screenshots, Recordings

review sandbox version, compared to mockup files in Monday task.

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _includes/news-and-events/podcasts.html
 - _site/assets/img/news-and-events/podcasts/3dprining.jpg
 - _site/news-and-events/podcasts/How-3D-Printing-is-Transforming-Health-Care/index.html
 - assets/img/news-and-events/podcasts/3dprining.jpg
 - news-and-events/podcasts/how-3d-printing-is-transforming-health-care.html
 - _includes/home/video.html
 - _site/assets/css/styles.css
 - _site/index.html
 - _site/news-and-events/index.html
 - assets/css/styles.css
 - news-and-events/index.html

### CODE REVIEW

**Comment by RyanCrux** [comment]
Link to homepage: https://www.dev-innovation.bgsandbox.com/oam/

Link to News & Events: https://www.dev-innovation.bgsandbox.com/oam/news-and-events/

**Review by ecstech-admin — CHANGES_REQUESTED** [review]
Looks great!

Add space between videos on home page in mobile view
image

**Comment by RyanCrux** [comment]
Good catch - all set here: 
Screenshot 2024-07-11 at 5 02 27 PM

File: _site/index.html
Comment (ecstech-admin) [inline _site/index.html:1]: Looks good at mobile-lg breakpoint but still getting spacing issue at mobile breakpoint.

Try changing "mobile-lg:margin-bottom-10" to "mobile:margin-bottom-10".

**Review by ecstech-admin — CHANGES_REQUESTED** [review]
Looks good at mobile-lg but still getting spacing issue at mobile breakpoint. Try changing "mobile-lg:margin-bottom-10" to "mobile:margin-bottom-10".

**Review by will-bbg — APPROVED** [review]

**Pull request merged**